src/libertem/io/dataset/base/tiling.py

```diff
@@ -582,7 +581,7 @@ def _get_size(self, io_max_size, udf, itemsize, partition, base_shape):
         Calculate the maximum tile size in bytes
         """
         udf_method = udf.get_method()
-        partition_size = itemsize * np.prod(partition.shape)
+        partition_size = itemsize * np.prod(partition.shape, dtype=np.uint64)
```

Comment:
`RuntimeWarning: overflow encountered in long_scalars` was thrown when I loaded the k2is dataset

Reply:
Good catch!